spec.bs

```diff
@@ -130,110 +129,65 @@ are:
     is not in the range [0, 2<sup>128</sup>−1].
 1. Throw {{RangeError}} if |contribution|["{{PAHistogramContribution/value}}"]
     is negative.
-1. [=list/Append=] |contribution| to the [=contributions cache=].
+1. Let |batchingScopeSteps| be the {{PrivateAggregation}} object's
+    [=PrivateAggregation/get batching scope steps=].
+1. If |batchingScopeSteps| is null, throw a new {{DOMException}} with name
```

Comment:
Can that happen? If not, it's better to assert it doesn't, rather than throwing

Reply:
Added a note here -- below we say "The global scope may wait to set the field to indicate that the API is not yet available."

Reply:
This is really just to prevent the API being available during the initial execution of a script (e.g. while `addModule()` runs). We only want the API to be available after that point. Definitely open to suggestions for better structuring this, but for now I've updated the description slightly to clarify.

Reply:
Added an example to better illustrate this, lmk if you think it's clear enough

Reply:
It's fine as far as the spec go, but worthwhile thinking about the developer ergonomics here:
* How would developers know when it's fine to run this API call and when it isn't? Is that part of our developer documentation?
* If I'm a library provider enabling folks to send reports, how should I write my code so that it never runs too early? (regardless of when callers may call it)